code/video/shapedetect.cpp

```diff
@@ -0,0 +1,203 @@
+//
+// Created by Beini Fang on 2018-01-25.
+// Detects and tags rectangles, triangles, and circles
+// Returns vertices of rectangles and triangles, and radius + center of circles
+//
+
+#include "shapedetect.h"
+
+#include "opencv2/core.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/objdetect.hpp"
+#include "opencv2/photo.hpp"
+
+#include <cmath>
+
+#include <iostream>
+#include <stdio.h>
+
+const int minTriangleArea = 10;
```

Comment:
It'd be nice if `ShapeDetect` could take these are arguments with these are default values